modules/acid/src/main/ts/ephox/acid/gui/components/SaturationBrightnessPalette.ts

```diff
@@ -104,7 +111,8 @@ const paletteFactory = (_translate: (key: string) => string, getClass: (key: str
       dom: {
         tag: 'div',
         attributes: {
-          role: 'presentation'
+          'role': 'presentation',
```

Comment:
This is still not fixed. If this is `presentation` I think the screen reader will just ignore any aria properties on it and thus this PR is not fixing what the ticket is about.

- [ ] Set proper slider role or something that the screen reader will pick up

Reply:
This ticket is actually not at all about this, it's about keyboard navigation explicitly for those who can see. The aria screenreader is something we're adding since we're changing things in the area. But sure, I can change that.

Reply:
But this would be the first thing we ever have in the UI that you can tab to that doesn't have any description at all in a screen reader. So if you are blind and accidentally tab to that thing you wouldn't know what it is. We need to think about the bigger picture not degrade a11y in one part while improving it in another part.

Reply:
No, I agree with you that it needs improvement, not arguing that at all. It is undeniably not as good as it should be. Or arguably 'good' at all. It just wasn't what the ticket is about ( and quite frankly this component isn't exactly accommodating either )